specs/beacon-chain.md

```diff
@@ -605,7 +611,11 @@ Extend the list of `AttestationRecord` objects in the `active_state` with those
 
 Let `proposer_index` be the validator index of the `parent.slot % len(get_shards_and_committees_for_slot(crystallized_state, parent.slot)[0].committee)`'th attester in `get_shards_and_committees_for_slot(crystallized_state, parent.slot)[0]`. Verify that an attestation from this validator is part of the first (ie. item 0 in the array) `AttestationRecord` object; this attester can be considered to be the proposer of the parent block. In general, when a block is produced, it is broadcasted at the network layer along with the attestation from its proposer.
```

Comment:
We should define this as `parent_proposer_index` and define `proposer_index` separately below.